types/feathersjs__authentication-jwt/feathersjs__authentication-jwt-tests.ts

```diff
@@ -0,0 +1 @@
+const dummy: any = null;
```

Comment:
In other contributions I've made it was often a sufficient test to import the main .d.ts and instantiate an instance of related classes. This forces tslint to flag invalid declarations.